components/TagList/TagList.js

```diff
@@ -1,21 +1,12 @@
 import { v4 as uuid4 } from "uuid";
 import { tagList } from "@/lib/db/db";
 import Tag from "../Tag";
-export default function TagList({ tags, editTag, activityId, tagId }) {
+export default function TagList({ tags }) {
   return (
     <ul>
       {tagList.map((tag) => {
-        const foundTag = tags?.find((id) => tag.id === id);
-        return (
-          <Tag
-            key={uuid4()}
-            activityId={activityId}
-            tagId={tagId}
-            editTag={editTag}
-            tag={tag}
-            foundTag={foundTag}
-          />
-        );
+        const foundTag = tags?.find((id) => tag.id == id);
+        return <Tag key={uuid4()} tag={tag} foundTag={foundTag} />;
```

Comment:
please do not use a `uuid()` you create in the loop as keys
```suggestion
        return <Tag key={tag.id} tag={tag} foundTag={foundTag} />;
```